File: src/targets/javascript/jquery/fixtures/text-plain.js
const settings = {
  async: true,
  crossDomain: true,
<<<<<<< HEAD
  url: 'https://httpbin.org/anything',
=======
  url: 'http://mockbin.com/har',
>>>>>>> upstream/master
  method: 'POST',
  headers: {
    'content-type': 'text/plain'
  },
  data: 'Hello World'
};

$.ajax(settings).done(function (response) {
  console.log(response);
});

File: src/targets/ruby/faraday/fixtures/text-plain.rb
require 'faraday'

conn = Faraday.new(
  url: 'https://httpbin.org',
  headers: {'Content-Type' => 'text/plain'}
)

response = conn.post('/anything') do |req|
  req.body = "Hello World"
end

puts response.status
puts response.body